src/ert/run_models/run_arguments.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class SingleTestRunArguments:
+    simulation_mode: str = field(init=False, default="Single test run")
+
+    active_realizations: List[bool]
+    current_case: str
+    target_case: str
+    iter_num: Optional[int] = 0
+    item_number: Optional[int] = 0
```

Comment:
I think this was a typo on my part and was supposed to be `iter_num`. (Which is why it's nice to have mypy to avoid dumb typos).